inc/classes/optimization/CSS/class-combine-google-fonts.php

```diff
@@ -121,20 +123,22 @@ protected function parse( $matches ) {
 				return;
 			}
 
-			$font = wp_parse_args( $query );
-
+			$font        = wp_parse_args( $query );
+			$font_family = $font['family'];
```

Comment:
It will match all links like this:
`<link(?:\s+(?:(?!href\s*=\s*)[^>])+)?(?:\s+href\s*=\s*([\'"])((?:https?:)?\/\/fonts\.googleapis\.com\/css(?:(?!\1).)+)\1)(?:\s+[^>]*)?>`

eg: `https://fonts.googleapis.com/css?`

If the matched URL has no query string, it will bailout and no google fonts will be combined.
However if it has query strings, I added a check for family and combined only those which have family included. The rest will be removed totally from HTML because are not valid (test: // Without Subsets & malformed URL (no family, but wrong query string) ).